messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -820,7 +820,8 @@ class MessagingTest : BaseMessagingTest() {
                     }
                     val eagerAttachment = dog.createAttachment(
                         eagerPlainTextFile,
-                        "text/plain"
+                        "text/plain",
+                        lazy = false
```

Comment:
The test wasn't actually testing this as an eager attachment, you have to specify lazy = fase.